CMakeLists.txt

```diff
@@ -161,6 +161,11 @@ include(SSGCommon)
 add_custom_target(validate)
 add_custom_target(stats)
 add_custom_target(profile-stats)
+add_custom_target(galaxy-ansible-roles)
+add_custom_command(
+    TARGET galaxy-ansible-roles POST_BUILD
+    COMMAND "${SSG_SHARED_UTILS}/update_galaxy_ansible_roles.py" --ssg-build-roles-directory "${CMAKE_BINARY_DIR}/roles/" --meta-template-path "${SSG_SHARED_UTILS}/galaxy_ansible_role_meta_template.yml"
+)
```

Comment:
this could be a separate script instead of being part of build. I wouldn't expect upload of anything to be in the build system.

Reply:
@mpreisler is your comment ^ all that would keep this PR from being merged